react-client/src/pages/plan/project-search/projectsearch.js

```diff
@@ -169,139 +179,155 @@ function ProjectSearch() {
+                  />
+                </div>
               </div>
-            </div>
 
-            <div className="dmpui-form-cols">
-              <div className="dmpui-form-col">
-                <TextInput
-                  label="Principle Investigator (PI)"
-                  type="text"
-                  required="required"
-                  name="principle_investigator"
-                  inputValue=""
-                  onChange={handleChange}
-                  id="principle_investigator"
-                  placeholder=""
-                  help="PI Names or Profile IDs, semicolon ';' separated"
-                  error=""
-                />
-              </div>
+              <div className="dmpui-form-cols">
+                <div className="dmpui-form-col">
+                  <TextInput
+                    label="Principle Investigator (PI)"
+                    type="text"
+                    required="required"
+                    name="principle_investigator"
+                    inputValue=""
+                    onChange={handleChange}
+                    id="principle_investigator"
+                    placeholder=""
+                    help="PI Names or Profile IDs, semicolon ';' separated"
+                    error=""
+                  />
+                </div>
 
-              <div className="dmpui-form-col">
-                <TextInput
-                  label="Award Year"
-                  type="text"
-                  required="required"
-                  name="award_year"
-                  inputValue=""
-                  onChange={handleChange}
-                  id="award_year"
-                  placeholder="Award Year"
-                  help="e.g. 2020"
-                  error=""
-                />
+                <div className="dmpui-form-col">
+                  <TextInput
+                    label="Award Year"
+                    type="text"
+                    required="required"
+                    name="award_year"
+                    inputValue=""
+                    onChange={handleChange}
+                    id="award_year"
+                    placeholder="Award Year"
+                    help="e.g. 2020"
+                    error=""
+                  />
+                </div>
               </div>
-            </div>
-          </form>
-        </div>
+            </form>
+          </div>
 
-        <div className="dmpdui-list project-list">
-          {projects.length === 0 ? (
-            <>
-              <div className="empty-list">
-                {!searching && (
-                  <p>Start searching to find your project…</p>
+          <h2 id="project-list-header" className="project-list-header">Search Results</h2>
+
+          <div className="table-container">
+            <div className="table-wrapper">
+              <table className="dmpdui-list-table dmpui-table  project-search-table" aria-live="polite" aria-labelledby="project-list-header">
```

Comment:
For @andrewebdev
I need to go to the CSS for this and remove the custom rules I created to highlight the selected grid row. Now that this changed into a table, those rules should be removed to reduce network payload (and potential other CSS related issues)